fedora_bot.py

```diff
@@ -97,6 +99,48 @@ def slack_notify(message: str):
     assert response.body == "ok"
 
 
+def get_open_pull_requests(args,component):
+    req = request.Request(f'https://src.fedoraproject.org/api/0/rpms/{component}/pull-requests', method="GET") # returns only open PRs by default
```

Comment:
This will merge ALL pull requests, right? I consider this dangerous, anyone can open a PR which might be actually malicious to our codebase, I think we need to distinguish PRs from packit tbh.

Reply:
Yeah, that's a good point, hadn't thought about that so much because nobody from our team ever opened a Fedora PR in the last few months. I'll look into filtering for packit PRs.